README.md

```diff
@@ -636,6 +654,27 @@ To check and install the Cyberith SDK, please follow the steps for Cyberith SDK
 ### Configuration
 To configure the Cyberith SDK please follow the steps for Cyberith SDK mentioned in [here](https://github.com/robotology/walking-teleoperation/blob/master/docs/Dependencies.md).
 
+## Xsens
+Support for this dependency is enabled by the `ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS` CMake option.
+
+**Warning: at the moment the Xsens SDK does not support macOS and Linux, so this option is only supported
+on Windows.**
+
+### System Dependencies
+To check and install the Xsens SDK, please follow the steps for Xsens SDK mentioned in [here](https://github.com/robotology/human-dynamics-estimation/wiki/Set-up-Machine-for-running-HDE#xsens-only-for-windows). 
+
+### Configuration
+To configure the Xsens SDK please follow the steps for Xsens SDK mentioned in [here](https://github.com/robotology/human-dynamics-estimation/wiki/Set-up-Machine-for-running-HDE#xsens-only-for-windows). 
+
+## Shoes
```

Comment:
I am confused by this dependency. Typically optional dependencies are meant as software external to the superbuild that we do not explicitly require nor we build with the superbuild, either because they are big and not available on all platforms (like Gazebo), or because they are proprietary (like most SDK). `FTShoes/FTSkShoes` is not another piece of software, and there is no related `ROBOTOLOGY_USES_FTSHOES` option. What is the rationale to discuss about a specific setup as a dependency?

Reply:
We should install the USB-CAN2 drivers, and add the necessary environment variables, which as far as I know it is external to superbuild.

Reply:
USB-CAN2 are different from ESD-CAN? Anyhow, in that case I would call this depending "USB-CAN2" or similar, in a way that it can be used also to enable other software (different from the FT Shoes) that depend on "USB-CAN2".

Reply:
Yes, as long as the `ROBOTOLOGY_USES_XSENS_MVN_SDK` option exists and is not enabled  automatically by `ROBOTOLOGY_USES_HUMAN_DYNAMICS` the wording is probably fine, sorry for the confusion in my reply.